ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -768,15 +768,12 @@ def arrays_to_numpy(x):
         frontend_fw = importlib.import_module(module_name)
         frontend_ret = frontend_fw.__dict__[fn_name](*args_frontend, **kwargs_frontend)
 
-        if ivy.isscalar(frontend_ret):
-            frontend_ret_np_flat = [np.asarray(frontend_ret)]
+        if frontend == "tensorflow" and isinstance(frontend_ret, tf.TensorShape):
+            frontend_ret_np_flat = [np.asarray(frontend_ret, dtype=np.int32)]
```

Comment:
Why is this not the case anymore for scalar values?

Reply:
The scalar case is being handled through the flatten_frontend_to_np call.